episodes/data-metadata.Rmd

```diff
@@ -0,0 +1,342 @@
+    **descriptive**, **administrative**, **structural**
+:::
+
+## Metadata records
+
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::: instructor
+## Lecture: Metadata in the scientific context
+
+**Slide set**: <a href="../files/slideSet/MetadataInScience_CarpentriesIncubator_ScientificMetadata.pdf" target="_blank" download>MetadataInScience.pdf</a>
+
+**Time estimate**: 10 min
+
+1. Now let's shift our focus on metadata annotation in the scientific context.
+2. I'm sure many of us have been in a similar situation. You start a new project that is based on the work of your predecessor or collaborators and you are supposed to start with reproducing the published results...
+3. You take a look at the accompnying publication and the data looks neat...
+4. But as soon as you open the underlying datasets, understanding the process on how this data was acquired is not so straight forward anymore. The variables might be hard to understand, there might be missing values, and the data cleaning is not necessarily apparent.
+5. So you take a look at the experiment's documentation...
+6. A study by Monya Baker, that was published in Nature in 2016 was able to show, that more than 70% of researchers have tried and failed to reproduce another scientis's experiments. And more than half have even failed to reproduce their own experiments.
+7. The worst practice that you can encounter in your scientific life is surely data, saved under an inexplicit file name with no documentation whatsoever.
+8. Slightly better practice are lab notes, which are still widely used as the go-to documentation method in experimental scientific areas. Using lab notes as documentation method, you can facilitate the findability... 
+9. by assigning meaningful file names to your data. However...
+10. lab notes come with various disadvantages. Lab notes are usually taken continuously and over the course of a 3-year contract, several hundred pages of lab notes can accumulate.
+11. so even though lab notes allow for some kind of documentation, they are largely unstructured and subjective, rarely following templates or guidelines. The continous note taking makes the documentation hard to retrieve. They are physically seperated from the data they describe and remain in the hand of the experimentator or institution hosting the experiments, which makes them hard to share. And let's admit it, handwriting can be a pain to read.
+12. One way to store your documentation with your data, is README-style metadata... 
+13. Just like taking lab notes, Readme-style files can be subjectively written natural text documents. 
+14. You can also find templates for Readme-style metadata records. Sticking to these templates will allow for increased structure in your documentation records. Now that we document our data digitally, we have increased options to link the metadata record to the described dataset, for example by naming them accordingly...
```

Comment:
```suggestion
14. You can also find templates for README-style metadata records. Sticking to these templates will allow for increased structure in your documentation records. Now that we document our data digitally, we have increased options to link the metadata to the described dataset, for example by naming them accordingly...
```
metadata *record* sounds like something specific, here I guess it just means "better way to interlink data and metadata", right?